libs/core/src/lib/grid-list/components/grid-list/grid-list.component.spec.ts

```diff
@@ -220,6 +224,31 @@ describe('GridListComponent', () => {
         expect(checkboxButtonsLength).toEqual(6);
     });
 
+    it('Multi Select mode: should throw selection events if unselected all', () => {
+        component.setMode('multiSelect');
+        spyOn(component, 'selectionChange');
+
+        fixture.detectChanges();
+
+        const radioButtons = fixture.debugElement.queryAll(By.css('.fd-grid-list__item .fd-checkbox'));
+        radioButtons[1].nativeElement.click();
+        radioButtons[2].nativeElement.click();
+        radioButtons[3].nativeElement.click();
+        radioButtons[4].nativeElement.click();
```

Comment:
How do we know that there is an element with index 4 for example?
Can we do it in a loop? Because as for me 1,2,3,4 - it is magical numbers

Reply:
The array of items itself is hardcoded